osclib/accept_command.py

```diff
@@ -51,6 +84,20 @@ def reset_rebuild_data(self, project):
         content = ET.tostring(root)
         http_PUT(url + '?comment=accept+command+update', data=content)
 
+    def virtually_accept_delete(self, request_id, package):
+        self.api.add_review(request_id, by_group=self.api.delreq_reivew)
```

Comment:
typo.. this should be self.api.delreq_review

Reply:
sigh...